src/python/espressomd/Makefile.am

```diff
@@ -58,6 +58,7 @@ PYXFILES =  \
 	magnetostatic_extensions.pyx \
 	particle_data.pyx \
 	reaction.pyx \
+	reaction_ensemble.pyx \
```

Comment:
Good catch. this should be removed here. I did not see that i commited it. After removal the python iterface will likely build.